services/web-app/components/svg-libraries/no-result/no-result.js

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright IBM Corp. 2022, 2022
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import { LogoGithub } from '@carbon/icons-react'
+import { Column, Grid } from '@carbon/react'
+import { Svg32Illustrator } from '@carbon-platform/icons'
+import clsx from 'clsx'
+import PropTypes from 'prop-types'
+
+import markdownStyles from '@/components/markdown/markdown.module.scss'
+import ResourceCard from '@/components/resource-card'
+import { mediaQueries, useMatchMedia } from '@/utils/use-match-media'
+
+import styles from '../svg-library.module.scss'
+
+const NoResult = ({
+  allIconResults,
+  setSelectedCategory,
+  selectedCategory,
+  type = 'icons',
+  pageName,
+  pageUrl
+}) => {
+  const designType = pageName === 'icon' ? 'an icon' : 'a pictogram'
+  const isSm = useMatchMedia(mediaQueries.sm)
+  return (
+    <div className={styles['no-result']}>
+      {allIconResults > 0 && (
+        <>
+          <span className={styles['search-label']}>
+            {`0 results found in the "${selectedCategory}" category.`}
+          </span>
+          <h2 className={markdownStyles.h2}>
+            {allIconResults} matches found in{' '}
+            <button
+              type="button"
+              onClick={() => setSelectedCategory(`All ${type}`)}
+              className={clsx(styles['all-svgs'], 'cds--link')}
+            >
+              all {type}.
+            </button>
+          </h2>
+        </>
+      )}
+      {!allIconResults && (
+        <>
+          <h2 className={markdownStyles.h2}>No result found</h2>
+          <p className={markdownStyles.paragraph}>
+            It appears we don’t have an icon that matches your search. Try different search terms or
+            give us a hand—submit your own design to the library!
+          </p>
+          <Grid className="resource-card-group" condensed={isSm}>
+            <Column sm={4} md={4} lg={6}>
+              <ResourceCard
+                subTitle={`Submit ${designType} design`}
+                href="https://github.ibm.com/brand/pictograms/issues/new"
+              >
+                <LogoGithub size={32} className={styles.icon} />
+              </ResourceCard>
+            </Column>
+            <Column sm={4} md={4} lg={6}>
```

Comment:
This should be `lg={4}` but its not working. Let me know if you want help digging into this weird one.